src/pages/agents.mdx

```diff
@@ -0,0 +1,76 @@
+# Secure interoperable agents
+
+AI is transforming consumer tech, with messaging becoming the main channel for interacting with agent services. This shift will scale message traffic astronomically, analogous to the web’s rise in the 2000s. Just as Cloudflare and https secured web traffic, messaging will need robust scalable end-to-end encrypted messages to protect sensitive information.
+
+## Risks
+
+Risks of not using end-to-end encryption for agent interactions exposes the users to what is called as Man in the **Middle Attacks**.
+
+> **Man in the Middle Attacks**: Intercept requests in between to alter or manipulate data sent or received by the AI service
+
+- **Phishing**: Messages can be intercepted and manipulated.
+- **Privacy**: Sensitive information read by unwanted parties
+- **Tampering**: Content can be altered without detection.
+
+:::tip
+More concrete sensitive data could include credit card details, private keys and passwords which is not yet widely spread but as agents become smarter more use cases will include this type of sharing.
+:::
+
+## XMTP
+
+XMTP provides end-to-end encrypted messaging for agent interactions, crucial for privacy, security, and compliance. Without it, messages are exposed to many security risks
+
+### Features:
+
+- **E2EE**: End to end MLS encryption
+- **Multi-agent**: Support multi-agent through group chats
+- **Interoperable**: Works across all platforms and frontends
+- **Trusted**: Decentralized, open source-
+- **Anonymous**: By default user identity is anonymous.
+
+## Frequently Asked Questions
+
+### 1. What are the risks of not using end-to-end encryption for agent interactions?
+
+These are some of the potential risks that agent messages are exposed in traditional web architectures.
+
+**1. Man-in-the-Middle (MitM) Attacks**
```

Comment:
TLS offers very good protection against MitM attacks. This feels like a bit of a straw-man argument, since no one is seriously suggesting running important agents over unsecured connections to a LLM service. I doubt any of the big LLM providers even allow unsecured connections.

Reply:
totally valid for big, trusted companies that control the entire user experience, including the frontend.

But thinking about a more distributed context, with many developers deploying agents, interacting with other agents,  APIs, servers, and frontends. TLS will likely be in use, but the trust of all these pieces cannot be guaranteed. E2EE could add an extra layer of protection, ensuring data is secure even if an endpoint is compromised.